dv/uvm/core_ibex/riscv_dv_extension/testlist.yaml

```diff
@@ -360,6 +369,7 @@
     +require_signature_addr=1
     +enable_debug_seq=1
     +enable_irq_multiple_seq=1
+    +disable_fetch_enable_seq=1
```

Comment:
What about the next test `riscv_assorted_traps_interrupts_debug_test` should we disable the fetch enable sequence for that as well?

Reply:
No need as that test doesn't have the same fragile, timeout prone, checking the others have. Though I've updated the timeouts so we don't need to disable the fetch enable sequence.